script/import_scripts/socialcast/test/test_data.rb

```diff
@@ -2531,12 +2531,12 @@
                         {
                             "id": 108,
                             "url": "https://demo.socialcast.com/attachments/108",
```

Comment:
This domain doesn't exist

Reply:
https://www.vmware.com/products/socialcast.html

Reply:
I'm not sure what to do about such things.

One approach is to suggest that someone else clean them up in some appropriate manner.

It's quite likely that shouldn't be me, since I can't realistically know how to replace such a thing, and also because even split this PR is still huge.

I can leave the fix in place, since it doesn't matter, or drop it.